packages/pycharm-community-bin/pycharm-community-bin.pacscript

```diff
@@ -0,0 +1,43 @@
+name="pycharm-community-bin"
+version="2021.3.3"
+pkgname="pycharm-community"
+gives="pycharm-community"
+maintainer="Emil Aarestrup <emil@aarestrup.nu>"
+description="Python IDE for professional developers aimed at productivity with a keyboard-centric focus."
+url="https://download.jetbrains.com/python/pycharm-community-${version}.tar.gz"
+hash="06ef4152d59b97da1186cdf288588d20d69a689fe75b2140130fa5d8326d07ad"
+
+desktop_files_path="$STOWDIR/$name/usr/share/applications"
+application_files_path="$STOWDIR/$name/usr/bin/$pkgname-$version"
+
+prepare() {
+true
+}
+
+build() {
+true
+}
+
+install() {
+
+sudo mkdir -p "$STOWDIR/$name" "$application_files_path" "$desktop_files_path"
+sudo cp -R . "$application_files_path"
+
+# generate .desktop file
+echo -e "[Desktop Entry]\n\
+Type=Application\n\
+Icon=$application_files_path/bin/pycharm.svg\n\
+Name=PyCharm Community Edition\n\
+Exec=$application_files_path/bin/pycharm.sh\n\
+Path=$application_files_path/bin\n\
+Terminal=false\n\
+Hidden=false\n\
+Categories=Development" | sudo tee "$desktop_files_path/PyCharm.desktop" >> /dev/null
```

Comment:
The `\n` doesn't need to be here, and move the desktop file to a separate file outside of the pacscript. Look [here](https://github.com/pacstall/pacstall-programs/tree/master/packages/librewolf-app) for guidance.